.github/ISSUE_TEMPLATE/request-a-self-review-check.yml

```
@@ -7,6 +7,7 @@ body:
   - type: markdown
     attributes:
       value: |
+        You can complete a self-review at any stage of the spec, but we recommend doing it before FPWD because considering internationalization from the beginning can lower the development cost.
```

I think this might work better broken into sentences. Also, I'm not sure that lowering the development cost is applicable to working groups. Perhaps:

```suggestion
        You can complete a self-review at any time.
        We recommend doing one as early as possible: even before FPWD, if possible.
        In our experience, early review helps avoid surprises,
        design changes, painful refactoring, 
        and other delays stemming from the horizontal review process
        when seeking advancement of the specification later.
```

---

Looks great. Thank you for the suggestion!